src/mavo.js

```diff
@@ -177,7 +177,18 @@ var _ = self.Mavo = $.Class({
 
 		this.permissions.can("login", () => {
 			// We also support a URL param to trigger login, in case the user doesn't want visible login UI
-			if (Mavo.Functions.url("login") !== null && this.index == 1 || Mavo.Functions.url(this.id + "-login") !== null) {
+			let loginUrlParam = null;
+			if (Mavo.Functions.url("login") !== null && this.index === 1) {
+				loginUrlParam = "login";
+			}
+			else if (Mavo.Functions.url(this.id + "-login") !== null) {
+				loginUrlParam = this.id + "-login";
+			}
+			if (loginUrlParam !== null) {
+				// Remove param from url
+				const currentURL = new URL(location.href);
+				currentURL.searchParams.delete(loginUrlParam);
+				history.replaceState(null, "", currentURL.toString());
```

Comment:
No need to call `toString()`, the argument is cast to a string anyway.

Reply:
I actually get a warning from my IDE that "Argument type URL is not assignable to parameter type string | null | undefined   Type URL is not assignable to type string"

Reply:
Ok, we can leave it as-is for now. What is your IDE?

Reply:
This is the checkstyle from IntelliJ Ultimate Edition

Reply:
Well, it appears that is a bug with your IDE. Check this out: https://github.com/whatwg/html/issues/6115